web3/_utils/transactions.py

```diff
@@ -219,7 +223,7 @@ def assert_valid_transaction_params(transaction_params: TxParams) -> None:
 
 
 def prepare_replacement_transaction(
-    w3: "Web3",
+    w3: Union["Web3", "AsyncWeb3"],
```

Comment:
The `prepare_replacement_transaction` doesn't use any async method, so I just reuse the sync version of `prepare_replacement_transaction`. I'm not sure if it is better to reuse it in this way or copy it to async_transactions.py.